http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -638,8 +640,18 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
         }
 
       // Save or retrieve a KubernetesCluster record for the app
+      // debugging
+      regionOpt = landingZoneResourcesOpt.map(_.region)
+      regionName = regionOpt.map(_.name).getOrElse("")
```

Comment:
In what condition would we get an empty string for the region? Would it make sense to leave a comment, or log a warning if the region does not exist?

Reply:
This is debugging code but I will answer this for where it is used around line 871.
 What is the best practice for checking for this? I changed it to a get() with a comment that it shouldn't be null so the get() will throw an exception if it is.

Reply:
Thanks! I would maybe query the dev or prod database to see if there are any apps without a region. If there is an expected case, then a comment should be fine, but if not, throwing an exception is the right thing to do imo

Reply:
Some of the unit tests have have apps without a region apparently (tests for when the user failed to complete something I think). And I am not suitable so don't think I can query prod. Let me know if the change I made looks right.

Reply:
Wouldn't this be true for any apps created on AKS before this code is released? Or am I missing something?